smart-contracts/package.json

```diff
@@ -21,7 +21,7 @@
         "@assemblyscript/loader": "^0.25.2",
         "@massalabs/as-types": "^2.0.0",
         "@massalabs/eslint-config": "^0.0.9",
-        "@massalabs/massa-as-sdk": "^2.5.2-dev",
+        "@massalabs/massa-as-sdk": "massalabs/massa-as-sdk#main",
```

Comment:
plese keep a proper version number here (not a branch). Its mandatory to handle breakin changes